src/Wallabag/CoreBundle/Resources/views/themes/material/Config/index.html.twig

```diff
@@ -196,6 +196,7 @@
                     </div>
 
                     <div id="set5" class="col s12">
+                        {% if app.user.config.taggingRules is not empty %}
```

Comment:
this test is here to remove useless lines when the user didn't create any tagging rule.